backend/src/main/java/com/flightsearch/backend/Services/AirlineInformationService.java

```diff
@@ -14,12 +14,12 @@
 @Service
 public class AirlineInformationService {
     @Autowired
-    AccessTokenService accessTokenService;
+    AccessTokenHandler accessTokenHandler;
 
     OkHttpClient client = new OkHttpClient();
 
     public String airlineNameLookUp(String airlineCode) throws IOException {
-        String token = accessTokenService.getAccessToken();
+        String token = accessTokenHandler.getAccessToken();
 
         HttpUrl.Builder urlBuilder = HttpUrl.parse("https://test.api.amadeus.com/v1/reference-data/airlines").newBuilder()
```

Comment:
Put API urls in application.properties file so it can be updated without recompiling the classes.
Also it is common to use multiple application-{env}.properties for multiple deployments: Dev, QA, UAT, Prod, etc.
You would want to use a different url when deploying your app to production.